docs/operations/database-backup.md

```diff
@@ -83,6 +83,39 @@ rsync -a data/*.0* backup
 2. Copy all files to the desired location.
 3. Create a copy of `chaser.chk` and call it `truncate.chk`. This effectively overwrites the restored `truncate.chk`.
 
+## Differential backup & restore
+
+The following procedure is designed to minimize the backup storage space, and can be used to do a full and differential backup.
+
+### Backing up
+
+Backup the index
+
+1. Make a list `indexFiles` of all the `index/<GUID>` and `index/<GUID>.bloomfilter` files in the source.
```

Comment:
@timothycoleman it seems that step 1 and 2 need to be swapped - otherwise if a merge occurs between step 1 and 2, the indexmap file can change before step 2 is executed and the original `indexFiles` list will not be accurate

Reply:
ah yes you're right. i think this change is necessary to be made since we are now comparing the contents of the index map file (instead of the content of the ptable listing like we were before)